### PR TITLE
fix: [CDS-33902]: Gear alignments for manifest details

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Summary:

Aligned gears for menifest details

##### Jira Links:

https://harness.atlassian.net/browse/CDS-33902

##### Screenshots:

Screenshot 2022-02-22 at 3 35 15 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger typecheck

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithHttp/HelmWithHttp.tsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ const HelmWithHttp: React.FC<StepProps<ConnectorConfigDTO> & HelmWithHttpPropTyp
                   />
                   {getMultiTypeFromValue(formik.values?.chartName) === MultiTypeInputType.RUNTIME && (
                     <ConfigureOptions
-                      style={{ alignSelf: 'center', marginBottom: 3 }}
+                      style={{ alignSelf: 'center', marginTop: 22 }}
```
Comment (devinder-harness) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithHttp/HelmWithHttp.tsx:182]: can we move this to classname ?

**Review by devinder-harness — APPROVED** [review]

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithHttp/HelmWithHttp.tsx [hunk 2]
```diff
@@ -179,7 +179,7 @@ const HelmWithHttp: React.FC<StepProps<ConnectorConfigDTO> & HelmWithHttpPropTyp
                   />
                   {getMultiTypeFromValue(formik.values?.chartName) === MultiTypeInputType.RUNTIME && (
                     <ConfigureOptions
-                      style={{ alignSelf: 'center', marginBottom: 3 }}
+                      className={css.gearStyle}
```
Comment (bhavya-sinha) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithHttp/HelmWithHttp.tsx:182]: This is breaking for all the manifest types. 
Also, i think `align-items: center` css should work fine.

**Review by satyam-harness — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good
please wait for @bhavya-sinha 's review
https://github.com/harness/harness-core-ui/pull/7449/files#r811837834

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithHttp/HelmWithHttp.tsx (hunk 2 above)
Comment (Sagar-Rajput7) [reply src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmWithHttp/HelmWithHttp.tsx]: Did the required changes for all the breaking points, will ask for others' review once you verify these changes.
Thanks

**Comment by Sagar-Rajput7** [comment]
retrigger jest

**Review by bhavya-sinha — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
LGTM

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

**Pull request merged**